.github/workflows/continuous-integration.yaml

```
@@ -36,3 +27,31 @@ jobs:
       - name: contract - test
         run: npm run test-ci
         working-directory: ./contract
+
+  prettier:
+    runs-on: ubuntu-latest
+    needs: tests
+    steps:
+    - uses: actions/checkout@v2
+
+    # run prettier, and also ensure any lock file changes are reverted
+    - name: Prettify web-client
+      run: yarn && yarn prettier && git restore yarn.lock
```

1. Let's use `yarn style` as ensure running both lint and style in correct order. Perhaps we should make sure it's called `yarn style` in all folders just for simplicity? Even if it's just an alias with prettier. 

2. Is there a case where yarn.lock will be changed? I believe running `yarn install` should never change the lock file? 🤔 

```suggestion
      run: yarn install && yarn style && git restore yarn.lock
```

---

I like the idea of "yarn style" -- changes made!

I thought I had seen a yarn.lock generated, but agreed, this shouldn't happen. So cleaned out those checks.